Tasks/PublishPipelineArtifactV1/task.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "ECDC45F6-832D-4AD9-B52B-EE49E94659BE",
+    "name": "PublishPipelineArtifact",
+    "friendlyName": "Publish pipeline artifact",
+    "description": "Publish a local directory or file as a named artifact for the current pipeline",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/publish-pipeline-artifact",
+    "helpMarkDown": "Publish a local directory or file as a named artifact for the current pipeline.",
+    "category": "Utility",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "groups": [],
+    "demands": [],
+    "preview": true,
+    "minimumAgentVersion": "2.140.1",
+    "inputs": [
+        {
+            "name": "artifactName",
+            "type": "string",
+            "label": "The name of this artifact",
+            "defaultValue": "drop",
+            "required": true,
+            "helpMarkDown": "The name of this artifact."
+        },
+        {
+            "name": "targetPath",
```

Comment:
"target" implies the opposite of "source". Can we rename this input to `path`?, but also allow `targetPath` and `pathToPublish` (assuming the system supports "aliases" for inputs). This will slightly simplify the migration from V0 or PBA.

Reply:
Also, `targetPath` on the Publish Build Artifact is the target UNC path, which makes it even more confusing to use this as the name for this input (which is the "source" path).

Reply:
We will discuss the potential field name change in the meeting this afternoon.

Reply:
Agreed to use `path` for the input name and to add `targetPath` as an alias (to help users migrating from PPA V0).